debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/ParallelSnapshotReader.java

```diff
@@ -147,12 +173,12 @@ public String name() {
     /**
      * A Halting Predicate for the parallel snapshot reader
      *
-     * This halting predicate assumes that there is another reader also running with a complementary halting predicate.
-     * The lagging reader's halting predicate is the only predicate that will return true, but it will signal for the
-     * other reader to halt at the same time.
+     * TODO: more documentation (that is correct)
      */
-
     /*package local*/ static class ParallelHaltingPredicate implements Predicate<SourceRecord> {
+
+        private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
`private static final Logger LOGGER = ...`

Reply:
I see a mix of `LOGGER` and `logger` in the code base, with a slight preference toward `logger` in general (and moreso in the mysql module). But I can change it to `LOGGER` in my new files if you like.

Reply:
Yes, we try to converge to make them static (thus `LOGGER`). I'll change this (and a few more stylistic things), as I'm merging this.